backend/main/models/community.go

```diff
@@ -180,10 +181,10 @@ const UPDATE_COMMUNITY_SQL = `
 	WHERE id = $21
 `
 const SEARCH_COMMUNITIES_SQL = `
-	SELECT id, name, body, logo, category	
+	SELECT id, name, body, logo, category, SIMILARITY(name, $1) as score	
 	FROM communities 
 	WHERE SIMILARITY(name, $1) > 0.1
-		AND category IS NOT NULL
+		AND category IS NOT NULL ORDER BY score DESC
```

Comment:
I don't think you can add the ORDER BY here because `addFiltersToSql` appends to this string

Reply:
Mmmmm right...